src/py/flwr/client/app.py

```diff
@@ -32,6 +33,7 @@ def start_client(
     server_address: str,
     client: Client,
     grpc_max_message_length: int = GRPC_MAX_MESSAGE_LENGTH,
+    root_certificates: Optional[str] = None,
```

Comment:
For my understanding, what's the reason for not using `Union[str, bytes]` (as it's done on the server-side)?

Reply:
```suggestion
    root_certificate: Optional[str] = None,
```